src/lib/features/feature-toggle/feature-strategies-read-model.ts

```diff
@@ -47,4 +46,60 @@ export class FeatureStrategiesReadModel implements IFeatureStrategiesReadModel {
               }
             : null;
     }
+
+    async getMaxConstraintValues(): Promise<{
+        feature: string;
+        environment: string;
+        count: number;
+    } | null> {
+        const rows = await this.db('feature_strategies')
+            .select(
+                'feature_name',
+                'environment',
+                'constraints',
+                this.db.raw(
```

Comment:
where do you use the constraint_count?

Reply:
Good catch: addressed in 02921837a0a83863507960d4e311296377f9b329